pkg/build/controller/controller_test.go

```diff
@@ -8,22 +8,49 @@ import (
 	kerrors "github.com/GoogleCloudPlatform/kubernetes/pkg/api/errors"
 
 	buildapi "github.com/openshift/origin/pkg/build/api"
+	builder "github.com/openshift/origin/pkg/build/builder"
 	buildtest "github.com/openshift/origin/pkg/build/controller/test"
 	imageapi "github.com/openshift/origin/pkg/image/api"
 )
 
-type okBuildUpdater struct{}
+type okBuildClient struct{}
 
-func (obu *okBuildUpdater) UpdateBuild(namespace string, build *buildapi.Build) (*buildapi.Build, error) {
+func (okc *okBuildClient) UpdateBuild(namespace string, build *buildapi.Build) (*buildapi.Build, error) {
 	return &buildapi.Build{}, nil
 }
 
-type errBuildUpdater struct{}
+// dummy methods to finish implementing the interface
+func (okc *okBuildClient) CreateBuild(namespace string, build *buildapi.Build) (*buildapi.Build, error) {
```

Comment:
This is the problem with this pattern - a bunch of empty methods that don't convey useful info to an implementor.  Which is why we don't do this...